UI/ChatGPT/README.md

```diff
@@ -1,99 +1,30 @@
-# ChatGPT - MVUX
+# ChatGPT
 
-## Guidance to use Records with MVUX
+This is a ChatGPT Sample app using the OpenAI SDK. The app was built using C# Markup for the UI and MVUX for state management, encouraging the flow of immutable data. Records are used to manage state and ensure immutability.
```

Comment:
@eriklimakc 

I would add an image of the project running here.
But it's just a suggestion. 
If you understand that it is not necessary, the content is already great.